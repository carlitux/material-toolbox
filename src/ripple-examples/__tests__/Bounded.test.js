// @flow
import React from 'react';
import { shallow } from 'enzyme';

import BoundedRipple from '../Bounded';

describe('example::BoundedRipple', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<BoundedRipple />);
    expect(component).toMatchSnapshot();
  });
});
