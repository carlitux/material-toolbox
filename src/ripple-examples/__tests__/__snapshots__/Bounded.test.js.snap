// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example::BoundedRipple Should match snapshoot 1`] = `
<Inner>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';
import Elevation from 'material-toolbox/elevation';

const component = (
<Elevation depth={2}>
  <Ripple>
    <div
      tabIndex=\\"0\\"
      className=\\"ripple-demo-surface\\">
      Interact with me!
    </div>
  </Ripple>
</Elevation>
)"
      title="Bounded"
    >
      <Elevation
        depth={2}
        transition={false}
      >
        <Ripple
          accent={false}
          cssOnly={false}
          disabled={false}
          primary={false}
          unbounded={false}
        >
          <div
            className="ripple-demo-surface"
            tabIndex="0"
          >
            Interact with me!
          </div>
        </Ripple>
      </Elevation>
    </Example>
  </Cell>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';
import Elevation from 'material-toolbox/elevation';

const component = (
<Elevation depth={2}>
  <Ripple cssOnly>
    <div
      tabIndex=\\"0\\"
      className=\\"ripple-demo-surface\\">
      Interact with me!
    </div>
  </Ripple>
</Elevation>
)"
      title="Bounded - CSS Only"
    >
      <Elevation
        depth={2}
        transition={false}
      >
        <Ripple
          accent={false}
          cssOnly={true}
          disabled={false}
          primary={false}
          unbounded={false}
        >
          <div
            className="ripple-demo-surface"
            tabIndex="0"
          >
            Interact with me!
          </div>
        </Ripple>
      </Elevation>
    </Example>
  </Cell>
</Inner>
`;
